src/dialogs/components/dialog/dialogField.html

```diff
@@ -0,0 +1,133 @@
+<div ng-if="vm.dialogField.visible" class="form-group" ng-class="{'has-error': vm.dialogField.fieldValidation===false}">
+  <label class="control-label col-md-2 col-lg-4 col-xl-2 col-sm-2">{{ ::vm.dialogField.label }}</label>
+  <div class="col-sm-5 col-lg-5" ng-if="vm.dialogField.fieldValidation===false">{{ vm.dialogField.errorMessage }}</div>
+    <div ng-switch on="vm.dialogField.type" class="col-sm-5 col-lg-5">
+      <input
+        ng-switch-when="DialogFieldTextBox"
```

Comment:
At least try to be consistent please :).

This file combines all three approaches to newlines...

You have `+<div ng-if="vm.dialogField.visible" class="form-group" ng-class="{'has-error': vm.dialogField.fieldValidation===false}">` which is long and not split into mulitple lines.

You have 
```diff
 +     <textarea
 +        ng-switch-when="DialogFieldTextAreaBox"
 +        ng-model="vm.dialogField.default_value"
 +        ng-disabled="vm.dialogField.read_only || vm.inputDisabled"
```

which is split before the first arg, and indented 2 chars for every arg.

And finally you have 

```diff
 +      <select ng-if="vm.dialogField.options.force_single_value"
 +              ng-switch-when="DialogFieldTagControl"
 +              ng-model="vm.dialogField.default_value"
```

where the newlines is after the first attribute, and the rest is indented beneath the start of the first.


Either approach would be fine, but all three in one file .. no ;).

Looking at the existing html in this projects, looks like the 3rd approach (break *after* first attribute, align under) is the right one.